qaframework-bdd-tests/package.json

```diff
@@ -24,7 +24,7 @@
     "refapp2Report": "mvn test -Dcucumber.filter.tags='@selenium and @report'",
     "refapp2RolesAndPrivileges":"mvn test -Dcucumber.filter.tags='@selenium and @rolesAndPrivileges'",
     "refapp2Person": "mvn test -Dcucumber.filter.tags='@selenium and @person'",
-    "refapp2PatientVisit": "mvn test -Dcucumber.filter.tags='@selenium and @visit'",
+    "refapp2PatientVisit": "mvn test -Dcucumber.filter.tags='@selenium and @patientVisit'",
```

Comment:
why renaming  to patient visit  ?

Reply:
@sherrif10  I put a comment on the ticket but you did not attend to it. It was about the same. With @visit, there was no test running so I am wondering how it got into master. Not until I changed it to @patientVisit that I got this test running.

Reply:
@Irenyak1 That's okay i went through it package.json file.that was as per design, then for visitNote test, we would want something different.